package.json

```diff
@@ -36,7 +36,7 @@
     "licenseCheck:run": "license-tool/run.sh"
   },
   "dependencies": {
-    "@eclipse-che/workspace-client": "0.0.1-1613117389",
+    "@eclipse-che/workspace-client": "^0.0.1-1613117389",
```

Comment:
Is it safe to use ^ here?
It means that newer incompatible version can be fetched, right?

Reply:
We agreed that ^ is better to have to let yarn resolve transitive dependencies in a right way.
Autoupgrade should not happen after version is fixed in yarn.lock.
The only thing that should be improved from our-end - we should introduce PR check that guarantees that no dependencies upgrade happens without the corresponding CQ check

Reply:
^version -  "Compatible with version", will update you to all future minor/patch versions, without incrementing the major version.
But it works before creating a yarn.lock file and will not be upgrading versions each time.